PowerUp/app/build.gradle

```diff
@@ -22,12 +21,14 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    buildToolsVersion '27.0.3'
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     testImplementation "junit:junit:$rootProject.jUnitVersion"
     implementation "com.android.support:appcompat-v7:$rootProject.supportLibraryVersion"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.0'
```

Comment:
Declare version as a variable in project level build.gradle file

Reply:
![constraint error](https://user-images.githubusercontent.com/26908195/40220180-d5b4c9d6-5a95-11e8-846b-4c301109c62a.png)
@m-murad Changing to variable is giving an error. Can you please help.